ipatests/test_integration/test_installation.py

```diff
@@ -170,6 +170,35 @@ def test_replica2_with_ca_kra_install(self):
     def test_replica2_ipa_dns_install(self):
         super(TestInstallWithCA1, self).test_replica2_ipa_dns_install()
 
+    def test_install_with_bad_ldap_conf(self):
+        """
+        Test a client install with a non standard ldap.config
+        https://pagure.io/freeipa/issue/7418
+        """
+        ldap_conf = paths.OPENLDAP_LDAP_CONF
+        base_dn = self.master.domain.basedn  # pylint: disable=no-member
+        client = self.replicas[0]
+        tasks.uninstall_master(client)
+        client.run_command(['ipa-client-install', '--uninstall', '-U'],
```

Comment:
`tasks.uninstall_client(client)` will do a job here. By the way, why you need to uninstall client separately? doesn't it handled by `ipa-server-install --uninstall`  in `tasks.uninstall_master(client)` ?

Reply:
good catch :) by mistake I forgot to delete that line